lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2271,11 +2273,12 @@ def scatter(self, xs, ys, zs=0, zdir='z', s=20, c=None, depthshade=True,
 
         if c is None:
             c = self._get_lines.get_next_color()
```

Comment:
when did we add cycling support directly into scatter3d? I would have let the 2D scatter handle that.